ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Block.hs

```diff
@@ -205,7 +205,7 @@ distribAnnTip AnnTip{..} =
     distrib (WrapTipInfo info) =
         AnnTip annTipSlotNo annTipBlockNo info
 
-undistribAnnTip :: SListI xs => NS AnnTip xs -> AnnTip (HardForkBlock xs)
+undistribAnnTip :: forall xs. SListI xs => NS AnnTip xs -> AnnTip (HardForkBlock xs)
```

Comment:
Redundant change

Reply:
It is needed for the `Index xs blk` in the `where` clause.